Framework/Algorithms/src/ReflectometryReductionOneAuto.cpp

```diff
@@ -592,8 +594,20 @@ bool ReflectometryReductionOneAuto::processGroups() {
     alg->setProperty("OutputWorkspaceWavelength", IvsLamName);
 
     // Handle transmission runs
-    if (firstTransG)
-      alg->setProperty("FirstTransmissionRun", firstTransG->getItem(i)->name());
+    auto transmissionRunSum = firstTransG->getItem(0)->name();
+    if (firstTransG) {
+      for (size_t item = 1; item < firstTransG->size(); item++) {
+        auto plusAlg = this->createChildAlgorithm("Plus");
+        plusAlg->setChild(true);
+        plusAlg->initialize();
+        plusAlg->setProperty("LHSWorkspace", transmissionRunSum);
+        plusAlg->setProperty("RHSWorkspace", firstTransG->getItem(item));
+        plusAlg->setProperty("OutputWorkspace", transmissionRunSum);
+        plusAlg->execute();
+        transmissionRunSum = plusAlg->getPropertyValue("OutputWorkspace");
+      }
+      alg->setProperty("FirstTransmissionRun", transmissionRunSum);
+    }
     if (secondTransG)
       alg->setProperty("SecondTransmissionRun",
```

Comment:
The second transmission run needs to be processed in the same way as the first tranmsission run